client/src/app/auth.guard.ts

```diff
@@ -18,11 +18,12 @@ export class AuthGuard implements CanActivate {
     const expectedRole = next.data.expectedRole;
     const token = localStorage.getItem('Authorization');
```

Comment:
no need to get token here . it is already written in jsonDecoder().

Reply:
removed

Reply:
remove this line